2.5/README.md

```diff
@@ -23,26 +23,111 @@ modules for their web applications. There is no guarantee for any specific npm o
 version, that is included in the image; those versions can be changed anytime and
 the nodejs itself is included just to make the npm work.
 
-Usage
+Usage in Openshift
 ---------------------
-For this, we will assume that you are using the `rhscl/ruby-25-rhel7 image`, available via `ruby:2.5` imagestream tag in Openshift.
+For this, we will assume that you are using the `ubi8/ruby-25 image`, available via `ruby:2.5` imagestream tag in Openshift.
 Building a simple [ruby-sample-app](https://github.com/sclorg/s2i-ruby-container/tree/master/2.5/test/puma-test-app) application
 in Openshift can be achieved with the following step:
 
     ```
-    oc new-app ruby:2.5~https://github.com/sclorg/s2i-ruby-container.git --context-dir=2.5/test/puma-test-app/
+    oc new-app ruby:2.5~https://github.com/sclorg/rails-ex.git
     ```
 
-The same application can also be built using the standalone [S2I](https://github.com/openshift/source-to-image) application on systems that have it available:
+**Accessing the application:**
+```
+$ oc get pods
+$ oc exec <pod> -- curl 127.0.0.1:8080
+```
 
-    ```
-    $ s2i build https://github.com/sclorg/s2i-ruby-container.git --context-dir=2.5/test/puma-test-app/ rhscl/ruby-25-rhel7 ruby-sample-app
-    ```
+Source-to-Image framework and scripts
+-------------------------------------
+This image supports the [Source-to-Image](https://docs.openshift.com/container-platform/4.4/builds/build-strategies.html#images-create-s2i_build-strategies)
+(S2I) strategy in OpenShift. The Source-to-Image is an OpenShift framework
+which makes it easy to write images that take application source code as
+an input, use a builder image like this Ruby container image, and produce
+a new image that runs the assembled application as an output.
```

Comment:
One more thing I've noticed (just a detail) is that the lines are manually wrapped, which is not needed for viewing for this is usually generated into html (from markdown).

Reply:
@pvalena I don't get your point, what you mean by that? To keep the lines' length 200 letters long?

Reply:
On the contrary - there's no need to insert new-lines manually, as a web page (or any markdown renderer) will do that for you.

Reply:
@pvalena right, renderer does the right thing, what I'm considering is the source of the README.md, the plain text -- are you suggesting to keep the lines long or do I misunderstand?

Reply:
Yes. But it's of no issue if we don't- it's just something inconsistent (unnecessary) I've noticed. It has no effect on the rendered document, so feel free to ignore my comment :).